tokens_transfer/tokens_transfer.circom

```diff
@@ -118,11 +123,12 @@ template Main(n,m) {
     token_type_from === token_type_to;
 
     // accounts updates
-    component new_hash_from = MultiMiMC7(4,91);
-    new_hash_from.in[0] <== pubkey_x;
-    new_hash_from.in[1] <== token_balance_from-amount;
-    new_hash_from.in[2] <== nonce_from+1;
-    new_hash_from.in[3] <== token_type_from;
+    component new_hash_from = MultiMiMC7(5,91);
+    new_hash_from.in[0] <== from_x;
+    new_hash_from.in[1] <== from_y;
+    new_hash_from.in[2] <== token_balance_from-amount;
```

Comment:
This is the refactoring I mentioned during the call `token_balance_from - amount`

Reply:
but from reading the conversation I see you tried not to touch the token_transfers hah